agent/unit-test/configstore_unittest.cpp

```diff
@@ -58,29 +64,31 @@ TEST_F(ConfigStoreTest, constructorEmptyStrOk) {
  * pointing at a parsable json file, should not throw an exception.
  */
 TEST_F(ConfigStoreTest, constructorFileOk) {
-    ASSERT_NO_THROW(BaseConfigStore(testDataDir + shortManifestPath));
+    ASSERT_NO_THROW(BaseConfigStore(buildPath(testDataDir, shortManifestPath)));
 }
 
 /* Constructing a BaseConfigStore with a file path,
  * pointing at a parsable file, should not throw an exception.
  */
 TEST_F(ConfigStoreTest, constructorFileOk2) {
-    ASSERT_NO_THROW(BaseConfigStore(testDataDir + manifestPath));
+    ASSERT_NO_THROW(BaseConfigStore(buildPath(testDataDir, manifestPath)));
 }
 
 /* Constructing a FilteredConfigStore with a directory path,
  * even if all files can not be parsed, should not throw an exception.
  */
 TEST_F(ConfigStoreTest, constructorDirNoJsonFiles) {
     // No Service Manifests, but ok dir
+    // TODO: Get rid of this + ""
```

Comment:
It was already there, it does not compile without it, and I did not want to spend time on that (because that isn't really part of this story).